timeless/access_control/manager_privileges.py

```diff
@@ -0,0 +1,39 @@
+import flask
+
+from timeless.access_control.methods import Method
+from timeless.employees.models import Employee
+
+
+def has_privilege(method=None, resource=None, *args, **kwargs) -> bool:
+    """Check if user with Manager role can access a particular resource."""
+    return __resources.get(resource, lambda *arg: False)(
+        method, *args, **kwargs
+    )
+
+
+def __employee_access(method=None, *args, **kwargs):
+    """
+    @todo #176:30min Add role to the employee model and check that Manager
+     can only access/modify employees that have role of master or
+     interns.
+    """
+    permitted, user = False, flask.g.get("user")
+    employee_id = kwargs.get("employee_id")
+    if user:
+        if employee_id:
+            permitted = check_employee(employee_id, method, user)
+        else:
+            permitted = True
```

Comment:
@krzyk why do we permit access if there is no ``employee_id``?

Reply:
@amihaiemil because it doesn't access particular employee, I followed the example in the owner_privileges here.
We don't have any endpoints for profile access yet so it is hard to guess.